fuzzers/044-clk-bufg-pips/build_zdb.py

```diff
@@ -0,0 +1,71 @@
+""" Tool for building zero db file for BUFG pips.
```

Comment:
Cannot this be done automatically instead of committing the `bits.dbf`?
Maybe by repeating the iterations until `build_zdb` succeeds. Mainly because there could be differences between different architectures that result in different `.zdb` outputs.

Reply:
Not easily, and we currently have no evidence that bits actually vary amoung the 7-series parts.

Reply:
Also this is roughly filed in https://github.com/SymbiFlow/prjxray/issues/347 , sort of.